### ISSUE TITLE
LaTeXStrings with Plots and (gr) not working in Jupyter

### ISSUE THREAD

**Issue by VivaldoMendes** [jheinen/GR.jl#404]
I have just upgraded Julia from 1.5.4 to 1.6.1. Apparently, LaTexStrings with Plots and (gr) are not working properly in Jupyter. In VSCode and Atom no problem at all. 

I am on a Windows 10 machine, 
```Julia 1.6.1
Plots v1.13.1
GR v0.57.4
LaTeXStrings v1.2.1
```

Last year, when upgrading from Julia 1.4 to 1.5 there was a similar problem with LaTeXStrings, issue [#2659](https://github.com/JuliaPlots/Plots.jl/issues/2659). As recommended in that issue, I have already done: `julia> ENV["GRDIR"] = ""` and `]build GR`, I also `]build Plots` and `]build LaTeXStrings`, but the problem persists.
 

MWE:
```
using Plots
using LaTeXStrings
x=randn(100);
plot(x, label = L"x_t")
```

I get the following error the first time I run the code above (the second time, the plot pops up but no label appears):

```
┌ Info: Precompiling Plots [91a5bcdd-55d7-5caf-9e0b-520d859cae80]
└ @ Base loading.jl:1317
┌ Warning: Package GR does not have Libdl in its dependencies:
│ - If you have GR checked out for development and have
│   added Libdl as a dependency but haven't updated your primary
│   environment's manifest file, try `Pkg.resolve()`.
│ - Otherwise you may need to report an issue with GR
└ Loading Libdl into GR from project dependency, future warnings for GR are suppressed.
┌ Warning: The call to compilecache failed to create a usable precompiled cache file for Plots [91a5bcdd-55d7-5caf-9e0b-520d859cae80]
│   exception = ErrorException("Required dependency GR [28b8d3ca-fb5f-59d9-8090-bfdbd6d07a71] failed to load from a cache file.")
└ @ Base loading.jl:1033
┌ Info: Precompiling GR [28b8d3ca-fb5f-59d9-8090-bfdbd6d07a71]
└ @ Base loading.jl:1317
┌ Warning: Package GR does not have Libdl in its dependencies:
│ - If you have GR checked out for development and have
│   added Libdl as a dependency but haven't updated your primary
│   environment's manifest file, try `Pkg.resolve()`.
│ - Otherwise you may need to report an issue with GR
└ Loading Libdl into GR from project dependency, future warnings for GR are suppressed.


```

**Comment by VivaldoMendes** [comment]
In Pluto, LateXStrings does not work with Plots and (gr) as well. The same code above leads to:

`LoadError: UndefVarError: @L_str not defined`

**Comment by jheinen** [comment]
I think that problem is not related to GR. Are you using different Julia versions or environments for Jupyter, VSCode, Atom, Pluto and the command line? I can't reproduce the problem.

**Comment by VivaldoMendes** [comment]
I think I have just one environment. For example, if I use the same MWE above with pgfplotsx() and Plots, it works. Everything was working OK before I upgraded Julia. I did the usual stuff with IJulia. I should emphasize that, currently, the only problem I have in Jupyter is with the label using the LaTeXStrings in Plots, (gr). If I remove the `L` string from the label, no problem at all with Plots. The pattern seems similar to what happened when upgrading from Julia 1.4 to 1.5.

**Comment by jheinen** [comment]
Then I'd assume that the following examples (without using `LaTeXStrings`] work, right?
```
using Plots
plot(randn(100), label="\$x_t\$")
```
or
```
using GR
plot(randn(100), labels=["\$x_t\$"])
```

**Comment by VivaldoMendes** [comment]
Thanks for your help. I don´t know why but for the first time (I migrated into Julia at v.1.0), I had a lot of problems of incompatibility between packages when upgraded from the previous version. For example, Flux collided with CSVFiles, QuantEcon with Distributions, and some others. I have managed to sort it out now. I think you were right: it had nothing to do with GR. Plotting with GR is working very well now.
